content/topics/web-frontend/_index.md

```diff
@@ -1,3 +1,19 @@
 ---
 title: Web Frontend
----
\ No newline at end of file
+ready: true
+---
+
+### Principles of design
+
+1.  Contrasts
+2.  Balance
+3.  Typography
+4.  Hierarchy
+5.  Repetition
```

Comment:
This topic doesn't make sense to me. Why does it exist?

We already have `topics/intro-to-web-design`, it would be better to just update that. And if that topic is sufficient then just use it as is.  

Let's keep our content DRY and cohesive